src/pooler.c

```diff
@@ -63,6 +63,8 @@ void cleanup_sockets(void)
 	struct ListenSocket *ls;
 	struct List *el;
 
+	int port;
+
```

Comment:
let's move this declartion to the `if` block where it's actually used to limit its scope.